KBKeyboardObserver/KBKeyboardObserver.h

```diff
@@ -48,6 +48,22 @@
  */
 - (void)keyboardObserver:(KBKeyboardObserver *)keyboardObserver observedKeyboardDidShowToRect:(CGRect)keyboardRect;
 
+
+/**
+ Called by the KBKeyboardObserver after the dismissal of the keyboard.
+ 
+ @param keyboardObserver observer instance
+ @param keyboardRect CGRect value that identifies the end frame of the keyboard in the reference view coordinates
+ */
+- (void)keyboardObserver:(KBKeyboardObserver *)keyboardObserver observedKeyboardDidChangeFrameToRect:(CGRect)keyboardRect duration:(NSTimeInterval)duration;
```

Comment:
The action here is `keyboardWillChangeFrame`, so maybe `observedKeyboardWillChangeFrameToRect:` is more appropriate?